### PR TITLE
Dragon api

### PR DESCRIPTION
**Pull request by Ol-create**
Fetch data from the Dragons endpoint  (https://api.spacexdata.com/v3/dragons) when a user navigates to the Dragons section.

Once the data are fetched, dispatch an action to store the selected data in Redux store:

- id
- name
- type
- flickr_images

- Follow the same logic as with the "Reserve dragon" - but you need to set the `reserved` key to `false`. 
- Dispatch these actions upon click on the corresponding buttons.

NOTE: Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation).

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import dragons from './dragons/dragon';
+
+// Combine all the reducers into one place
+const rootReducer = combineReducers({
+  // rockets,
+  // missions,
```
Comment (amrendrakind) [inline src/redux/configureStore.js:8]: - [ ] Please remove commented code as it is not a good practice from a clean code context. :wink:

File: src/redux/dragons/dragon.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import fetchDragons from './api';
+
+const FETCHDRAGON = 'FETCHDRAGON';
+const RESERVEDRAGON = 'RESERVEDRAGON';
+const UNRESERVEDRAGON = 'UNRESERVEDRAGON';
```
Comment (amrendrakind) [inline src/redux/dragons/dragon.js:5]: - [ ] [OPTIONAL]  Consider using :duck:  duck pattern as recommended :smile:

**Review by amrendrakind — CHANGES_REQUESTED** [review]
Hi @Ol-create    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

**Comment by Ol-create** [comment]
hi @amrendrakind thank you for your effort.
The comment has been resolved

**Review by amrendrakind — APPROVED** [review]
### Status: Approved :100: 

Hi @Ol-create 👋🏾,

image

Your project is complete! 👏🏾
There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! tada tada

Top Highlights 💪🏻
:heavy_check_mark: Good use of Pull Request
:heavy_check_mark: No linter errors.
:heavy_check_mark: You used the correct Git-flow.
:heavy_check_mark: Commit messages are meaningful.
:heavy_check_mark: The app works as expected
:heavy_check_mark: Good quality of code

Cheers and Happy coding! :clap: :clap: :clap: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @amrendrakind in your question so I can receive the notification.

**Pull request merged**